test/jdk/com/sun/crypto/provider/Cipher/AEAD/GCMBufferTest.java

```diff
@@ -62,7 +67,9 @@
     int[] sizes;
     boolean incremental = false;
     // In some cases the theoretical check is too complicated to verify
-    boolean theoreticalCheck;
+    boolean theoreticalCheckUpdate; // This variable is never set to true in order to disable
+                                    // length verification after cipher.update for GCM Buffer Tests
+    boolean theoreticalCheckFinal;
```

Comment:
Please put comments before declarations (following the existing code):
```
    // In some cases the theoretical check is too complicated to verify
    boolean theoreticalCheckFinal;
    // This variable is never set to true in order to disable
    // length verification after cipher.update for GCM Buffer Tests
    boolean theoreticalCheckUpdate;
```

Reply:
Done.